### ISSUE TITLE
Why did you augment the test set as well as the training set? And why did that boost the performance of your proposed method?

### ISSUE THREAD

**Issue by rgeng5** [rfuruta/pixelRL#14]

**Comment by rfuruta** [comment]
Are you talking about this implementation? If so, you might be misunderstanding because test-time augmentation is not implemented in this code.

If you are talking about our paper, please see Sec. 7.2 in this report (https://openaccess.thecvf.com/content_cvpr_2017_workshops/w12/papers/Agustsson_NTIRE_2017_Challenge_CVPR_2017_paper.pdf). We just followed this augmentation technique. The performance was improved by ensembling the enhanced predictions.
